presto-hive/src/main/java/com/facebook/presto/hive/metastore/HivePrivilegeInfo.java

```diff
@@ -103,6 +104,23 @@ public boolean isContainedIn(HivePrivilegeInfo hivePrivilegeInfo)
                         (!isGrantOption() && hivePrivilegeInfo.isGrantOption())));
     }
 
+    public PrivilegeInfo toPrivilegeInfo()
```

Comment:
Didn't get you. Are you suggesting to get rid of the HivePrivilegeInfo class? I think it's better to keep that class since it contains methods such as `parsePrivilege` that are specific to Hive connector.